src/Plugin.php

```diff
@@ -59,6 +59,10 @@ public static function init() {
     add_action('rss2_item', __NAMESPACE__ . '\Feed::rss2_item');
 
     UserFrontend::init();
+
+    add_filter('the_content', __CLASS__ . '::the_content');
+    add_filter('wp_get_attachment_metadata', __CLASS__ . '::wp_get_attachment_metadata', 10, 2);
+    add_filter('post_thumbnail_html', __CLASS__ . '::removeSvgWidthHeightAttributes', 10, 3);
```

Comment:
1. 👍 
2. It's not named like the filter, because it's also used from within the Admin class. So I thought it would make more sense to have a more specific method name.